wazimap_ng/datasets/admin/dataset_admin.py

```diff
@@ -62,7 +62,7 @@ class DatasetAdmin(DatasetBaseAdminModel):
         }),
         ("Dataset Imports", {
             "fields": (
-                "import_dataset", "imported_dataset",
+                "content_type", "import_dataset", "imported_dataset",
```

Comment:
Could you also add a filter for content type?

Reply:
Done 👍